src/content/docs/en/guides/cms/cloudcannon.mdx

```diff
@@ -9,6 +9,125 @@ i18nReady: false
+
+The important things to configure here are:
+
+- Any required environment variables
+- Install Command: Command to install dependencies before build. By default this runs `npm install`. This is run as a line of bash and can be anything you want. A common change here is to use `yarn install` instead.
+- Build Command: Command to build your site. By default this runs `npm run build`. This is similar to the install command. An example of this build is `astro build`.
+- Output Path: Path to the output folder you are building into. This is important because CloudCannon needs to know which folder to treat as the static output of your site. By default Astro builds into the `dist` folder.
+
+You may find you need extra commands to complete your build. A common example is using postcss; for this you can use build hooks, adding specially named bash scripts that run at different steps in the build. For more information, see [CloudCannon’s build hook documentation](https://cloudcannon.com/documentation/articles/extending-your-build-process-with-hooks/).
+
+### Preview URL
+
+Once your site is built, you will see a big green tick on the screen. At this point, your site’s files are synced and a successful build has been run. This means you have unlocked a newly generated preview URL.
+
+In the top left corner of the screen under your site name, there is a blue link, styled *adjective-noun.cloudvent.net*. Clicking this link will open a new tab to the hosted site on CloudCannon. 
+
+It is a helpful step at this point to check that the site looks correct. If it does, you can move on to editor configuration. If you need to update your build, this can be done in your settings.
+
+Your new *adjective-noun.cloudvent.net* site will give you a preview after every change. Use this to show off your new site.
+
+## Configuring
+
+Astro has relatively strong opinions about how to organize the content in your site. This means that CloudCannon's editing concepts integrate naturally with Astro. CloudCannon organizes data using two main concepts:
+
+- *Data*: These are standalone data files. This is ideal for site configuration or files that don’t need to repeat. Data is edited by altering to a single file.
+- *Collections*: A folder of files that represent a repeatable data format. This is a good fit for pages, blogs, staff members, recipes, etc. Collections are edited by altering any file or adding more files.
+
+In Astro terms, CloudCannon Collections correspond to Astro [Content Collections](/en/guides/content-collections/). Data files, on the other hand, correspond to standalone JSON files that are imported with Javascript. CloudCannon supports a set of file formats for files in Data or Collections:
+
+- Structured data files: JSON, YAML, TOML, CSV, TSV.
+- Markup files: HTML, Markdown.
+- Combination files: HTML with front matter, Markdown with front matter.
+
+### Tell CloudCannon about your data
```

Comment:
See my comment above. Again, there are no steps here, although the instruction to "define your collections" is a good one! That's where we can totally use the "The example below defines a collection containing..." trick!